src/meta_schedule/integration.cc

```diff
@@ -20,6 +20,8 @@
 #include <tvm/relay/function.h>
 #include <tvm/tir/function.h>
 
+#include "./utils.h"
```

Comment:
Do we allow relative path?

Reply:
Yep we allow include relative path for headers internal under `src/`